src/cryptography/x509/base.py

```diff
@@ -576,6 +576,21 @@ def get_attribute_for_oid(self, oid: ObjectIdentifier) -> bytes:
 CertificateSigningRequest.register(rust_x509.CertificateSigningRequest)
 
 
+class Policy(metaclass=abc.ABCMeta):
```

Comment:
Why does this need to be an ABC?

Reply:
No reason, this was another re-export thing. Will remove.

Reply:
ed83cd639